### PR TITLE
task224

### PR DESCRIPTION
**Pull request by mertsalovaa**

### CODE REVIEW

File: Projects/TEAM-2/Mertsalova_Iryna/Tasks/Task224/Program.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    class Program
+    {
+        //Дано масив випадкових чисел в діапазоні від -20 до +20.
+        //Необхідно знайти позиції крайніх негативних елементів
+        //(самого лівого негативного елементу і самого правого негативного елементу)
+        //і впорядкувати елементи, що знаходяться між ними.
+
+        static void Main(string[] args)
+        {
+            Random rnd = new Random();
```
Comment (LoveMyDaughter) [inline Projects/TEAM-2/Mertsalova_Iryna/Tasks/Task224/Program.cs:18]: Never used.

**Review by LoveMyDaughter — COMMENTED** [review]
Great!

File: Projects/TEAM-2/Mertsalova_Iryna/Tasks/Task224/Program.cs (hunk 1 above)
Comment (mertsalovaa) [reply Projects/TEAM-2/Mertsalova_Iryna/Tasks/Task224/Program.cs]: Fixed.

**Review by romanukhaav — APPROVED** [review]
Good job!

**Pull request merged**